INSTALL

```diff
@@ -61,6 +61,11 @@ Strongly recommended:
   * chardet character encoding detection library
       (https://chardet.github.io/)
 
+For htpasswd support on standalone server:
+
+  * passlib package
+      (https://pypi.org/project/passlib/) 
+
```

Comment:
This top-most list of dependencies is really reserved for the _the very most_ important recommendations.  Let's move your addition here down into the "ViewVC Standalone Server" section (lines 465 ff).